tgui/packages/tgui/interfaces/PreferencesMenu/antagonists/antagonists/sentiencepotionspawn.ts

```diff
@@ -0,0 +1,23 @@
+import { Antagonist, Category } from '../base';
+import { multiline } from 'common/string';
+
+const SentientCreature: Antagonist = {
+  key: 'sentiencepotionspawn',
+  name: 'Sentient Creature',
+  description: [
+    multiline`
+		Either by cosmic happenstance, or due to crew's shenanigans, you have been
+		given sentience!
+	  `,
+
+    multiline`
+		This is a blanket preference. The more bening ones include random human 
+		level intelligence events, the cargorilla, and creatures uplifted via sentience 
+		potions. The less friendly ones include the regal rat, and the boosted 
+		mining elite mobs. A mixed case is mobs travelling on emergency shuttles.
```

Comment:
i dont understand what this means

Reply:
This role is applied to multiple roles, like the pai role, so I wanted to list all of the roles it currently applies to. Perhaps I should remove this description, so the list does not have to be maintined, and perhaps, make a completely separate role for each of these creatures in this PR, instead of adding to the problem.

Reply:
Github confused me with the highlights, is this about the whole block, or about the last bit about travelling on emergency shuttles? The latter is referring to the sentience fun balloons, which gives sentience to the monsters on the "Nar Nar" shuttle, or the Bardrone and Barmaid on the bar shuttle.